releasenotes/notes/fix-adder-gates-39cf3d5f683e8880.yaml

```diff
@@ -0,0 +1,20 @@
+---
+features_transpiler:
+  - |
+    Added :class:`.FullAdderSynthesisDefault` plugin that chooses the best decomposition
+    for :class:`.FullAdderGate` based on the number of clean ancilla qubits 
+    available. 
+  - |
+    Improved :class:`.HalfAdderSynthesisDefault` and :class:`.ModularAdderSynthesisDefault`
+    plugins for :class:`.HalfAdderGate` and :class:`.ModularAdderGate` respectively,
+    choosing the best decomposition based on the number of clean ancilla qubits available.
```

Comment:
If we want to include this in 1.3.1 these should be bugfixes, otherwise these improvements would have to be split off for 2.0 (these were likely oversights from before and valid as bugfix... 😉)

Reply:
I have rephrased this as fixes ;). And indeed I actually do think now that including one missing default plugin and fixing the logic of the other default plugins should be considered as fixes.